src/ScaleUtil/ScaleUtil.js

```diff
@@ -45,13 +49,17 @@ class ScaleUtil {
       axisDomain[1] = Math.max(ext[1], axisDomain[1]);
     }
 
+    if (axis.harmonize) {
+      axisDomain[0] = Math.pow(10, Math.floor(Math.log(axisDomain[0]) / Math.log(10)));
+      axisDomain[1] = Math.pow(10, Math.ceil(Math.log(axisDomain[1]) / Math.log(10)));
+    }
     if (axis.scale === 'log' && (axisDomain[0] === 0 || axisDomain[1] === 0 ||
       isNaN(axisDomain[0]) || isNaN(axisDomain[1]))) {
       if (axisDomain[0] === 0 || isNaN(axisDomain[0])) {
-        axisDomain[0] = Number.MIN_VALUE;
+        axisDomain[0] = 0.000001;
```

Comment:
Maybe this is a candidate for constant field value defined at the beginning?

Reply:
Done, collecting refinements in https://github.com/terrestris/d3-util/pull/6